### ISSUE TITLE
Titanic example fails to fit

### ISSUE THREAD

**Issue by jdelmerico** [EpistasisLab/tpot#1147]
I am trying to run the code example for the Titanic example (https://github.com/EpistasisLab/tpot/blob/master/tutorials/Titanic_Kaggle.ipynb) as-is and receive the following error:

Generation 1 - Current best internal CV score: -inf
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
~/.local/share/virtualenvs/jupyter-Og1NxUOc/lib/python3.8/site-packages/tpot/base.py in fit(self, features, target, sample_weight, groups)
    729                 warnings.simplefilter('ignore')
--> 730                 self._pop, _ = eaMuPlusLambda(
    731                     population=self._pop,

~/.local/share/virtualenvs/jupyter-Og1NxUOc/lib/python3.8/site-packages/tpot/gp_deap.py in eaMuPlusLambda(population, toolbox, mu, lambda_, cxpb, mutpb, ngen, pbar, stats, halloffame, verbose, per_generation_function, log_file)
    280         if per_generation_function is not None:
--> 281             per_generation_function(gen)
    282 

~/.local/share/virtualenvs/jupyter-Og1NxUOc/lib/python3.8/site-packages/tpot/base.py in _check_periodic_pipeline(self, gen)
   1051         """
-> 1052         self._update_top_pipeline()
   1053         if self.periodic_checkpoint_folder is not None:

~/.local/share/virtualenvs/jupyter-Og1NxUOc/lib/python3.8/site-packages/tpot/base.py in _update_top_pipeline(self)
    837                         break
--> 838                 raise RuntimeError('There was an error in the TPOT optimization '
    839                                    'process. This could be because the data was '

RuntimeError: There was an error in the TPOT optimization process. This could be because the data was not formatted properly, or because data for a regression problem was provided to the TPOTClassifier object. Please make sure you passed the data to TPOT correctly. If you enabled PyTorch estimators, please check the data requirements in the online documentation: https://epistasislab.github.io/tpot/using/

During handling of the above exception, another exception occurred:

RuntimeError                              Traceback (most recent call last)
<ipython-input-39-1c4b83193ef6> in <module>
      1 tpot = TPOTClassifier(verbosity=2, max_time_mins=2, max_eval_time_mins=0.04, population_size=40)
----> 2 tpot.fit(titanic_new[training_indices], titanic_class[training_indices])

~/.local/share/virtualenvs/jupyter-Og1NxUOc/lib/python3.8/site-packages/tpot/base.py in fit(self, features, target, sample_weight, groups)
    771                     # raise the exception if it's our last attempt
    772                     if attempt == (attempts - 1):
--> 773                         raise e
    774             return self
    775 

~/.local/share/virtualenvs/jupyter-Og1NxUOc/lib/python3.8/site-packages/tpot/base.py in fit(self, features, target, sample_weight, groups)
    762                         self._pbar.close()
    763 
--> 764                     self._update_top_pipeline()
    765                     self._summary_of_best_pipeline(features, target)
    766                     # Delete the temporary cache before exiting

~/.local/share/virtualenvs/jupyter-Og1NxUOc/lib/python3.8/site-packages/tpot/base.py in _update_top_pipeline(self)
    836                                                     error_score="raise")
    837                         break
--> 838                 raise RuntimeError('There was an error in the TPOT optimization '
    839                                    'process. This could be because the data was '
    840                                    'not formatted properly, or because data for '

RuntimeError: There was an error in the TPOT optimization process. This could be because the data was not formatted properly, or because data for a regression problem was provided to the TPOTClassifier object. Please make sure you passed the data to TPOT correctly. If you enabled PyTorch estimators, please check the data requirements in the online documentation: https://epistasislab.github.io/tpot/using/

## Process to reproduce the issue

[ordered list the process to finding and recreating the issue, example below]

1. Run Titanic example in Jupyter notebook

## Expected result

1. Succesful completion of call to fit()

**Comment by weixuanfu** [comment]
I ran a quick test in [Google colab](https://colab.research.google.com/gist/weixuanfu/a9ba804167031eb4dbbf37fedd1f93ff/issue1147.ipynb) and please check it. This issue was not reproduced. 

But I think `max_eval_time_mins=0.04` (about 2 seconds) maybe too small to run the demo in some environment. Please remove it from `TPOTClassifier` and have another try.

**Comment by jdelmerico** [comment]
Thanks @weixuanfu for the quick reply!  I tried both removing it and increasing it with the same result locally.  I also increased the max_time_mins parameter as a test.   

It does run in Colab, but not locally in my Juypter notebook.  I also download the ipynb file from Colab and tried it.  Same result, but sometimes I received a "failed to converge after 200 iterations" message also.  TPOT version on my local linux version is the same as Colab.  I suppose it could be environment otherwise so closing but thank you again for the reply.
